packages/component-library/styles/type.scss

```diff
@@ -11,30 +11,30 @@ $ca-weight-medium: 500;
 $ca-weight-semibold: 500; // Note: in Sketch, semibold is 600. But murmur has an existing value of semibold=500 that is heavily used.
 
 // Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
-$ca-default-font-family: "Open Sans", Helvetica, Arial, sans-serif;
-$ca-default-font-base-size: 0.875rem; /* 14px */
-$ca-default-font-descender-height: 0.115;
+$ca-inter-font-family: "Inter", sans-serif;
```

Comment:
Do we definitely not want any fall back fonts (Helvetica, Arial) here? Genuinely asking, I'm not sure whether just `sans-serif` is good enough nowadays?

Reply:
I'm not sure to be honest! I created this Kaizen issue a little while back to do more research: https://github.com/cultureamp/kaizen-design-system/issues/829. This seemed "good enough" for now.

Reply:
Adding "Noto Sans" does seem important though. I will make this change to and at least keep it the same as the tokens.
 
I realise I can just use on tokens, but I didn't want to have the dependency (and get into potential dependency issues). The real goal is to get rid of the `ca-` stuff entirely.